Explorer/Assets/Scripts/Global/Dynamic/PortableExperiences/PortableExperiencesController.cs

```diff
@@ -0,0 +1,218 @@
+        private readonly IScenesCache scenesCache;
+
+        private readonly RealmData realmData = new ();
+
+        private readonly ObjectProxy<World> globalWorldProxy;
+        private List<Entity> entitiesToDestroy = new ();
+        public Dictionary<string, Entity> PortableExperienceEntities { get; } = new ();
+        private World world => globalWorldProxy.Object;
+        private List<IPortableExperiencesApi.SpawnResponse> spawnResponsesList = new ();
+
+        public PortableExperiencesController(
+            ObjectProxy<World> world,
+            IWeb3IdentityCache web3IdentityCache,
+            IWebRequestController webRequestController,
+            IScenesCache scenesCache)
+        {
+            globalWorldProxy = world;
+            this.web3IdentityCache = web3IdentityCache;
+            this.webRequestController = webRequestController;
+            this.scenesCache = scenesCache;
+        }
+
+        public async UniTask<IPortableExperiencesApi.SpawnResponse> CreatePortableExperienceAsync(string ens, string urn, CancellationToken ct, bool isGlobalPortableExperience = false)
+        {
+            //According to kernel implementation, the id value is used as an urn
+            //https://github.com/decentraland/unity-renderer/blob/b3b170e404ec43bb8bc08ec1f6072812005ebad3/browser-interface/packages/shared/apis/host/PortableExperiences.ts#L28
+            //And is validated first. As it has no ipfs config, it uses the one from the current realm apparently
+
+            string worldUrl = string.Empty;
+
+            if (!string.IsNullOrEmpty(urn))
+            {
+                //TODO: Enable loading PX from urns using current scene realm data. -> will be done in next iteration.
+                //worldUrl = IpfsHelper.ParseUrn(urn).BaseUrl.Value;
+            }
+
+            if (EnsUtils.ValidateEns(ens)) { worldUrl = EnsUtils.ConvertEnsToWorldUrl(ens); }
+
+            if (!worldUrl.IsValidUrl()) throw new ArgumentException("Invalid Spawn params. Provide a URN or an ENS name");
+
+            var portableExperiencePath = URLDomain.FromString(worldUrl);
+            URLAddress url = portableExperiencePath.Append(new URLPath("/about"));
+
+            await UniTask.SwitchToMainThread();
+
+            GenericDownloadHandlerUtils.Adapter<GenericGetRequest, GenericGetArguments> genericGetRequest = webRequestController.GetAsync(new CommonArguments(url), ct, ReportCategory.REALM);
+
+            ServerAbout result = await genericGetRequest.OverwriteFromJsonAsync(serverAbout, WRJsonParser.Unity);
+
+            if (result.configurations.scenesUrn.Count == 0)
+            {
+                //The loaded realm does not have any fixed scene, so it cannot be loaded as a Portable Experience
+                throw new Exception($"Scene not Available in provided Portable Experience with ens:{ens} - urn: {urn}");
+            }
+
+            realmData.Reconfigure(
```

Comment:
Would a `realmData` support many portable experiences? What if the `about` url changes?

Reply:
Nope, I was supposed to create a new realmData for each PX and this slipped the checks...